tests/device_tests/cardano/test_sign_tx.py

```diff
@@ -102,26 +106,84 @@ def test_cardano_sign_tx_with_multiple_chunks(client, parameters, result):
 
     expected_responses = [
         messages.PassphraseRequest(),
-        # XXX as many ButtonRequests as paginations. We are relying on the fact that
-        # there is only one fixture whose pagination is known.
-        # If that changes, we'll need to figure out something else.
+        *[messages.CardanoTxItemAck()] * 4,
```

Comment:
I'm not a 100% convinced this is a good idea. It looked a bit simpler before rebasing on top of the `ButtonRequest` changes. But perhaps it's worth it to have such a test and it won't probably change too often.

Reply:
The original purpose of this test was to ensure that the response is properly chunked. I don't think that applies anymore, so feel free to drop the test altogether.

Reply:
I basically deleted the test but in the end I decided to move it to `sign_tx.json` - so there would be a test which tests a transaction with many of the parameters set. The communication checks are gone though.